news/2018-02-20-beatmap-spotlights-january-2018.md

```diff
@@ -0,0 +1,132 @@
+
+Being the only nominated taiko mapset with no X-rated difficulty, it stands out from the rest through its appropriate use of various mapping techniques. [yuzu__rinrin](https://osu.ppy.sh/u/4616752) is flexible on his style - while showing the consistent drum-based logic when the music gets more vibrant, he is also open to pitch-oriented hitsounding as a balance during calmer parts. Such style is accompanied by the proper density and SV changes which act as both great presentation to players and nice interpretation of the music.
+
+Despite the short song length, it is a well-made mapset that shows his expertise in taiko mapping, as well as understanding of musical emotion. Definitely worth mentioning in this month's Spotlights!
+
+## osu!catch
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/algebra.jpg)](https://osu.ppy.sh/beatmapsets/468281)
+
+Let's welcome [Spectator](https://osu.ppy.sh/users/702598) once again, to bring his masterpiece to the top of our osu!catch monthly Spotlights! The map Function Phantom - Algebra gives us an excellent interpretation of how an inspiring technical map can be. Featured as a map in CWC 2017 Semifinals NoMod mappool, the Overdose, collaborated by Spectator and [Minato Yukina](https://osu.ppy.sh/users/531253), impresses us with its creative flow for the magic on strong Hyperdashes toward standstill buzz sliders. Besides, [alienflybot](https://osu.ppy.sh/users/636114) and [Sinnoh](https://osu.ppy.sh/users/4236057) also provide us with nice answers of how to combine convenience and innovation together into a map neatly. Good job!
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/force-of-ra.jpg)](https://osu.ppy.sh/beatmapsets/573479)
+
+[Deif](https://osu.ppy.sh/users/318565) is mapping again! Similar to his previous work, Memme - Force of Ra screams of his own personal mapping style: irregular patterns, gimmicky direction change with strong Hyperdashes, strong consistency across. Moreover, the Overdose stands out for its genius operating of Hyperdashes in kiai time. There are also many interesting patterns in Rain, Platter, and easier difficulties, rounding the whole mapset in a super balanced spread. Be sure to check this out, especially if you're a fan of Deif style maps.
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/marigold.jpg)](https://osu.ppy.sh/beatmapsets/605710)
+
+[Spectator](https://osu.ppy.sh/users/702598)'s fine art for Deemo lovers! Collaborating with [Ascendance](https://osu.ppy.sh/users/2931883) and [Sinnoh](https://osu.ppy.sh/users/4236057), Spectator brings us Marigold by M2U feat. Guriri, an appealing song with a top quality map. Despite the relatively low BPM, the intensity of the song is well expressed with wigglings and bursts used right to the point. On top of the map itself, it contains a splendid storyboard adding to its excellence. Be sure to check out this masterpiece!
```

Comment:
Missing Ascendance in the banner unless we dont count people that map the low diffs

Reply:
Nope, my brain just died. Fixed it.